docs/open5gs-and-ueransim/README.md

```diff
@@ -0,0 +1,30 @@
+# Open5gs Container Image
```

Comment:
Update this document with info deploying  open5gs together with ueransim (currently only open5gs)